stable/gcloud-sqlproxy/templates/deployment.yaml

```
@@ -20,8 +20,8 @@ spec:
       annotations:
 {{ toYaml .Values.podAnnotations | indent 8 }}
     spec:
-      {{- if .Values.rbac.create }}
-      serviceAccountName: {{ include "gcloud-sqlproxy.fullname" . }}
+      {{- if .Values.serviceAccount.create }}
+      serviceAccountName: {{ include "gcloud-sqlproxy.serviceAccountName" . }}
```

```suggestion
      serviceAccountName: {{ template "gcloud-sqlproxy.serviceAccountName" . }}
```

nit: Also you can remove the if statement around `serviceAccountName: {{ template "gcloud-sqlproxy.serviceAccountName" . }}` .... as per templates generated when using `helm create` today